Nasal/c172p.nas

```diff
@@ -194,6 +194,22 @@ var switches_save_state = func {
 };
 
 ##########################################
+# Thermometer
+##########################################
+
+var thermometer_switch = func {
+    var val = getprop("controls/switches/thermometer-switch");
+    var test = 1 + val;
```

Comment:
You should be able to replace these two lines (202 and 203) with:

```javascript
var test = math.mod(1 + val, 3);
```

Even better is to directly put the result in line 204 (inline it)